e2e/utils.go

```diff
@@ -486,3 +496,76 @@ func validatePVCAndAppBinding(pvcPath, appPath string, f *framework.Framework) {
 		Fail(err.Error())
 	}
 }
+
+func validateNormalUserPVCAccess(pvcPath string, f *framework.Framework) {
+	pvc := loadPVC(pvcPath)
+	pvc.Namespace = f.UniqueName
+	pvc.Name = f.UniqueName
+	framework.Logf("The PVC  template %+v", pvc)
+	err := createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
+	if err != nil {
+		Fail(err.Error())
+	}
+	var user int64 = 2000
+	app := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "pod-run-as-non-root",
+			Namespace: f.UniqueName,
+			Labels: map[string]string{
+				"app": "pod-run-as-non-root",
+			},
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:    "write-pod",
+					Image:   "alpine",
+					Command: []string{"/bin/sleep", "999999"},
+					SecurityContext: &v1.SecurityContext{
+						RunAsUser: &user,
+					},
+					VolumeMounts: []v1.VolumeMount{
+						{
+							MountPath: "/target",
+							Name:      "target",
+						},
+					},
+				},
+			},
+			Volumes: []v1.Volume{
+				{
+					Name: "target",
+					VolumeSource: v1.VolumeSource{
+						PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
+							ClaimName: pvc.Name,
+							ReadOnly:  false},
+					},
+				},
+			},
+		},
+	}
+
+	err = createApp(f.ClientSet, app, deployTimeout)
+	if err != nil {
+		Fail(err.Error())
+	}
+
+	opt := metav1.ListOptions{
+		LabelSelector: "app=pod-run-as-non-root",
+	}
+	execCommandInPod(f, "echo testing > /target/testing", app.Namespace, &opt)
```

Comment:
Dont we need an error check here ? @Madhu-1

Reply:
@humblec error check already done here https://github.com/ceph/ceph-csi/blob/9d27b6b425c0a63d8b664047dfd1a9172cbbfe2f/e2e/utils.go#L128-L129